When I compile:browser at webpack,got a error [Can't resolve 'stream']

# Error

```text
ERROR in ./src/utils/data.ts 9:14-31
Module not found: Error: Can't resolve 'stream' in '~***\bee-js\src\utils'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "stream": require.resolve("stream-browserify") }'
        - install 'stream-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "stream": false }
resolve 'stream' in '***\bee-js\src\utils'
  Parsed request is a module
  using description file: E:\EWork\bee_work\fork_projects\bee-js\package.json (relative path: ./src/utils)
    resolve as module
     ***\bee-js\src\utils\node_modules doesn't exist or is not a directory
      ***\bee-js\src\node_modules doesn't exist or is not a directory
      looking for modules in ***\bee-js\node_modules
        single file module
          using description file:***\bee-js\package.json (relative path: ./node_modules/str
eam)
            no extension
              ***\bee-js\node_modules\stream doesn't exist
            .ts
             ***\bee-js\node_modules\stream.ts doesn't exist
            .js
              ***\bee-js\node_modules\stream.js doesn't exist
        ***\bee-js\node_modules\stream doesn't exist
      ***\node_modules doesn't exist or is not a directory
      ***\node_modules doesn't exist or is not a directory
     ***\node_modules doesn't exist or is not a directory
      E:\node_modules doesn't exist or is not a directory
 @ ./src/bee.ts 24:12-35
 @ ./src/index.ts 39:11-27

```

## ENV

  - window 10
  - node 14.15.1
  - webpack 5.37.1
  - webpack-cli 4.7.0
  - bee-js 0.12.0 branch

--- a/webpack.config.ts
+++ b/webpack.config.ts
@@ -76,6 +76,7 @@ const base = async (env?: Partial<WebpackEnvParams>): Promise<Configuration> => 
       fallback: {
         path: false,
         fs: false,
+        stream: false,
       },
     },
     optimization: {

